index.html

```diff
@@ -233,21 +409,22 @@ <h1>About me</h3>
             <div class="form-group">
                 <input class="input" type="text" name="name" maxlength="30" placeholder="Full name" rows="10" required>
                 <input class="input" type="email" name="email" placeholder="Email address" required>
-                <textarea name="send" maxlength="500" required placeholder="Write me something"></textarea>
+                <textarea name="send" maxlength="399" required>Write me something</textarea>
                 <input class="submit" type="submit" value="Get in touch">
             </div>
         </form>
     </section>
-    <hr>
+    <hr class="hr">
     <footer class="footer">
         <div class="icon">
             <ul>
                 <li><a href="https://github.com/BlessmanNewton"><i class="fa fa-github" aria-hidden="true"></i></a></li>
-                <li><a href="https://linkedin.com/BlessmanNewton"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
-                <li><a href="https://angellist.com/BlessmanNewton"><i class="fa fa-angellist" aria-hidden="true"></i></a>
-                </li>
-                <li><a href="https://twitter.com/BlessmanNewton"><i class="fa fa-twitter" aria-hidden="true"></i></i></a>
+                <li><a href="https://linkedin.com/BlessmanNewton"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
                 </li>
+                <li><a href="https://angellist.com/BlessmanNewton"><i class="fa fa-angellist"
+                            aria-hidden="true"></i></a></li>
+                <li><a href="https://twitter.com/BlessmanNewton"><i class="fa fa-twitter"
+                            aria-hidden="true"></i></i></a></li>
```

Comment:
Kindly work on your `Get In touch` button , and increase the font-size of the text to the left so that it is similar to the design you were given in the figma template 👍 

Your design :

![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/2bcc313d-4cd6-4863-9f9a-58e03750b477)

Figma design :
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/453fed72-2936-45a7-855e-19e4c90bb602)